package.json

```diff
@@ -4,7 +4,12 @@
     "start:frontend": "cd frontend && ng serve",
     "build:frontend": "cd frontend && npm run build",
     "build:server": "cd server && npm run build",
-    "build": "npm run build:server && npm run build:frontend"
+    "build": "npm run build:server && npm run build:frontend",
+    "install:ui": "cd frontend && npm install",
```

Comment:
I'd propose to be consistent about calling it "`ui`", not a mix of sometimes "frontend" sometimes "ui". Same for the root folder, `frontend/` → `ui/`!

Reply:
changing it to ui to be more consistent